src/pages/MainScreen.jsx

```diff
@@ -18,6 +19,7 @@ export const MainScreen = () => {
           <Routes>
             <Route path='accounts' element={<CheckingAccountsList/>}/>
             <Route path='creation' element={<CheckingAccountCreation/>}/>
+            <Route path='registration' element={<Registration/>}/>
```

Comment:
These are common path examples you can use:

/checking-account/create

/user/create